packages/taquito-rpc/src/types.ts

```diff
@@ -954,6 +954,21 @@ export type RunCodeResult = {
   big_map_diff?: ContractBigMapDiff;
 };
 
+export type RPCRunViewParam = {
+  contract: string;
+  entrypoint: string;
+  input: MichelsonV1Expression;
+  chain_id: string;
+  source?: string;
+  payer?: string;
+  gas?: BigNumber;
+  unparsing_mode?: UnparsingModeEnum;
```

Comment:
"unparsing_mode" is required by the RPC if I am not mistaken. If so, I think we should make this property required in the type.